apps/sms/style/sms.css

```diff
@@ -902,6 +902,18 @@ section[role="region"].new .article-list[data-type="list"]  li {
   pointer-events: none;
 }
 
+/* margin bottom adjustment for inline-block bdi element */
+.suggestion > .name {
```

Comment:
I think we should cover contact prompt header ellipsis (one from contact_renderer.js) in this patch as well. What do you think?